components/fieldgroup/stories/fieldgroup.mdx

```diff
@@ -0,0 +1,79 @@
+import { ArgTypes, Canvas, Meta, Description, Title } from '@storybook/blocks';
+
+import * as FieldGroupStories from './fieldgroup.stories';
+
+<Meta of={FieldGroupStories} title="Docs" />
+
+<Title of={FieldGroupStories} />
+<Description of={FieldGroupStories} />
+
+## Variants
+
+### Vertical
+
+A vertical group of fields:
```

Comment:
I tried to pull in information from the .stories.js file wherever possible in this MDX page. My thought was: if it's informational, put it in the .stories.js file, if it's organizational only, like this description, it can be written out in the MDX.